src/lib/components/Alert/Alert.scss

```diff
@@ -15,16 +13,8 @@
         color: $white;
       }
     }
-
     &::before {
       filter: invert(1);
     }
   }
 }
-
-.ldaf-alert--site-wide.usa-alert--info .usa-alert__body .usa-link {
-  color: $primary-light;
-  &:visited {
-    color: $primary-light;
-  }
-}
```

Comment:
Link color is now set via the settings. Talked with Courtney and he said it should be fine to show the visited state on alerts (with the caveat that we want to keep the link white on the site-wide emergency alert).